### ISSUE TITLE
Accessibility: Ctrl+Up/DownArrows does not work in terminal created in editor area

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#21467]
CC -meganrogge

Ctrl+Up/DownArrow (command history) does not work in terminal buffer created in the editor area.
